### DISCUSSION TITLE
Update 6/13/2022 Update 2

### DISCUSSION THREAD

**Discussion by natehome** [FlipperMaker/flippermaker.github.io#10] [Updates]
## Update 6/13/2022 Update 2
* Support for NTAG213, NTAG215, NTAG216! Increased URL lengths! Create NTAG21X NFC files that open a URL.

**Comment by Mastroo59** [comment]
> ## Update 6/13/2022 Update 2
> * Support for NTAG213, NTAG215, NTAG216! Increased URL lengths! Create NTAG21X NFC files that open a URL.
